.github/workflows/bump.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Automatic version updates'
+on:
+  schedule:
+    # minute hour dom month dow (UTC)
+    - cron: '0 16 * * *'
+  # enable manual trigger of version update
+  workflow_dispatch:
+
+jobs:
+  version_update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: wader/bump/action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.BUMP_TOKEN }}
```

Comment:
Can this not use the automatically supplied `secrets.GITHUB_TOKEN`?
https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Reply:
Yes sorry, was a note to myself to note forgot to do it.

Reply:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token

Reply:
Thanks for the reminder. That is certainly less than ideal in this case. Oh well!